app/models/card.rb

```diff
@@ -1,4 +1,6 @@
 class Card < ActiveRecord::Base
+  scope :created_before, -> (data) { where('review_date <= ?', data) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.